api/config/locales/es.yml

```diff
@@ -64,6 +64,7 @@ es:
     lines: Líneas
     config:
       title: Ajustes
+      data: Data
```

Comment:
"Datos" in Spanish

Reply:
Also, maybe the "data" item should go always in the same position inside the `config` section, in all files? I mean, it could be second, or third, but it should be the same in all files.